### PR TITLE
Fix psql location

### PR DESCRIPTION
**Pull request by faveoled**

### CODE REVIEW

File: org.pgadmin.pgadmin4.yml [hunk 1]
```diff
@@ -118,7 +118,7 @@ modules:
         -xf -
       - cp -r usr/* ${FLATPAK_DEST}/
       - mv ${FLATPAK_DEST}/pgadmin4 ${FLATPAK_DEST}/pgAdmin4
-      - cat config_distro.py usr/pgadmin4/web/config_distro.py > ${FLATPAK_DEST}/pgAdmin4/web/config_distro.py
+      - cat usr/pgadmin4/web/config_distro.py config_distro.py > ${FLATPAK_DEST}/pgAdmin4/web/config_distro.py
```
Comment (razzeee) [inline org.pgadmin.pgadmin4.yml:121]: Why the change in order?

Comment (faveoled) [reply org.pgadmin.pgadmin4.yml:121]: distro config overrides some of the defaults

**Comment by faveoled** [comment]
image
the change has been done. psql works for me now
